src/components/ListItem.js

```diff
@@ -0,0 +1,27 @@
+import { useProducts } from '../hooks/useProducts';
+
+const ListItem = () => {
+  const { products, loading, error } = useProducts();
+
+  const showProducts = () => {
+    if (products.length === 0) {
+      return <p>You don't have any product in your list</p>;
+    }
+
+    return products.map(({ id, name, date }) => (
+      <li key={id}>{`Product: ${name} => Created at: ${new Date(
+        date,
+      ).toLocaleString()}`}</li>
+    ));
```

Comment:
I'd suggest to wrap this with an `<ul>` element to have valid markup.

Currently the output would be:
```
<div>
  <li>...</li>
</div>
```